OWNERS

```diff
@@ -2,25 +2,19 @@
 
 approvers:
 - jduimovich
-- sbose78
 - zregvart
 - simonbaird
-- michkov
 - johnmcollier
 - jgwest
 - bamachrn
-- amfred
-- gorkem
 - rhopp
 - alexeykazakov
 - chmouel
 - dirgim
 - elsony
 - gazarenkov
 - skabashnyuk
-- deboer-tim
 - ksprinkl
 - scoheb
 - eisraeli
 - amisstea
-- bkorren
```

Comment:
Seems wrong. Barak still part of program

Reply:
Barak is a member of redhat-appstudio using a different Github ID: `ifireball`

Reply:
Do you think I should change it to `ifireball` instead of only removing `bkorren`?

Reply:
I suggest to change to ifireball